pvlib/irradiance.py

```diff
@@ -994,6 +1006,103 @@ def king(surface_tilt, dhi, ghi, solar_zenith):
     return sky_diffuse
 
 
+def muneer(surface_tilt, surface_azimuth, dhi, ghi, dni_extra, b=5.73,
+           solar_zenith=None, solar_azimuth=None, projection_ratio=None):
+    '''
+    Determine sky diffuse irradiance on a tilted surface using the
+    Muneer model.
+
+    This Muneer transposition model is described in [1]_.
+
+    Parameters
+    ----------
+    surface_tilt : numeric
+        Surface tilt angles in decimal degrees. ``surface_tilt`` must
+        be >=0 and <=180. The tilt angle is defined as degrees from horizontal
+        (e.g. surface facing up = 0, surface facing horizon = 90)
+
+    surface_azimuth : numeric
+        Surface azimuth angles in decimal degrees. ``surface_azimuth``
+        must be >=0 and <=360. The azimuth convention is defined as degrees
+        east of north (e.g. North = 0, South=180 East = 90, West = 270).
+
+    dhi : numeric
+        Diffuse horizontal irradiance. [W/m^2]
+
+    ghi : numeric
+        Global horizontal irradiance in W/m^2.
+
+    dni_extra : numeric
+        Extraterrestrial normal irradiance in W/m^2.
+
+    b : numeric, default 5.73
+        Radiance distribution index, introduced by Moon and Spencer [2]_
+        to model luminance distribution of overcast sky. [unitless]
+        Recommended values from [1]_:
+
+           - isotropic: b = 0
+           - shaded surface: b = 5.73 (default)
+           - sunlit surface, overcast sky: b = 1.68
+           - sunlit surface, non-overcast sky: b = -0.62
+
+    solar_zenith : numeric
+        Solar apparent (refraction-corrected) zenith angles in decimal
+        degrees. Must supply ``solar_zenith`` and ``solar_azimuth`` or
+        supply ``projection_ratio``.
+
+    solar_azimuth : numeric, optional
+        Solar azimuth angles in decimal degrees. Must supply
+        ``solar_zenith`` and ``solar_azimuth`` or supply
+        ``projection_ratio``.
+
+    projection_ratio : numeric, optional
+        Ratio of angle of incidence projection to solar zenith angle
+        projection. Must supply ``solar_zenith`` and ``solar_azimuth``
+        or supply ``projection_ratio``.
+
+    Returns
+    -------
+    poa_sky_diffuse : numeric
+        In-plane diffuse irradiance from the sky. [W/m^2]
+
+    References
+    ----------
+    .. [1] Muneer, T., 1990. Solar radiation model for Europe. Building
+       Services Engineering Research and Technology 11, 153-163.
+       :doi:`10.1177/014362449001100405`
+
+    .. [2] Moon, P., Spencer, D.E., 1942. Illumination from a non-uniform sky.
+       Trans. Illum. Eng. Soc. (London) 37, 707-725.
+       :doi:`10.1177/096032719302500301`
```

Comment:
```suggestion
    .. [2] Moon, P., Spencer, D.E., 1942. Illumination from a non-uniform sky.
       Trans. Illum. Eng. Soc. (London) 37, 707-725.
```
I don't think that the 10.1177/096032719302500301 DOI is correct. It links to [this publication](https://journals.sagepub.com/doi/10.1177/096032719302500301?icid=int.sj-abstract.citing-articles.75), not the Moon, P. paper cited here in the references. I do not think the Moon (1942) paper has a DOI.
The first DOI for Muneer (1990) looks fine though.

Reply:
you are completely right, I tried to look the DOI but I put one that referenced the one from 1942. My bad...